app/controllers/users_controller.rb

```diff
@@ -145,6 +145,7 @@ def onboarding_update
     if params[:user]
       permitted_params = %i[summary location employment_title employer_name last_onboarding_page]
       current_user.assign_attributes(params[:user].permit(permitted_params))
+      current_user.profile_updated_at = Time.current
```

Comment:
This is probably technically not necessary for _brand new users_, but I tossed it in to make sure cache clears as normal when content updated.